frontend/src/apis/food.ts

```diff
@@ -3,9 +3,11 @@ import { GetFoodListReq, GetFoodListRes } from '@/types/food/remote';
 import { client } from '.';
 
 export const getFoodList = async ({ keyword }: GetFoodListReq) => {
-  const { data } = await client.get<GetFoodListRes>(
-    `/api/v1/foodList?keyword=${keyword.join(',')}`,
-  );
+  const { data } = await client.get<GetFoodListRes>('/pet-foods', {
+    params: {
+      keyword: keyword.length ? keyword.join(',') : undefined,
```

Comment:
에어비앤비 컨벤션을 참고한 아래 방식은 어떨까요?
```suggestion
      keyword: Boolean(keyword.length) ?? keyword.join(',')
```

Reply:
Boolean으로 감싸주는 것 좋네요! 반영했습니다~